Bottom: 9aaa3745471db8be1f14ee2a3f0a5b97b1670a21
Top:    f4af0c95643d8cfd240a83a0398c1bff9d3e85a8
Author: Igor Stoppa <igor.stoppa@huawei.com>
Date:   2018-07-30 09:10:06 +0300

Refresh of fix-rare-write

---

diff --git a/include/linux/rare_write.h b/include/linux/rare_write.h
index 6a458a380312..d7398f8c59a8 100644
--- a/include/linux/rare_write.h
+++ b/include/linux/rare_write.h
@@ -13,6 +13,7 @@
 #ifndef _LINUX_RARE_WRITE_H
 #define _LINUX_RARE_WRITE_H
 
+#include <linux/compiler.h>
 #include <linux/mm.h>
 
 extern long __start_rare_write_after_init;
